extensions/api/data-management/contractdefinition/src/test/java/org/eclipse/dataspaceconnector/api/datamanagement/contractdefinition/service/ContractDefinitionServiceImplTest.java

```diff
@@ -43,7 +44,9 @@ class ContractDefinitionServiceImplTest {
     private final ContractDefinitionStore store = mock(ContractDefinitionStore.class);
     private final ContractDefinitionLoader loader = mock(ContractDefinitionLoader.class);
     private final TransactionContext transactionContext = new NoopTransactionContext();
-    private final ContractDefinitionServiceImpl service = new ContractDefinitionServiceImpl(store, loader, transactionContext);
+    private final ContractDefinitionObservable observable = mock(ContractDefinitionObservable.class);
```

Comment:
I think there should be a test verifying that the `observable` actually got invoked